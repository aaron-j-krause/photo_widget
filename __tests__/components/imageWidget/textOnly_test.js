import React from 'react'
import renderer from 'react-test-renderer'
import TextOnly from '../../../src/components/imageWidget/textOnly'

test('Should render description as text', () => {
  const rendered = renderer.create(<TextOnly description="test" />)

  expect(rendered).toMatchSnapshot()
})
