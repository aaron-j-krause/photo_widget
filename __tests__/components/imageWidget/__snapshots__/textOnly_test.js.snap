exports[`test Should render description as text 1`] = `
<h4
  className={undefined}>
  test
</h4>
`;
